### PR TITLE
docs: crawlee / apify-sdk-js links in actor readmes

### PR DESCRIPTION
**Pull request by barjin**

### CODE REVIEW

File: packages/actor-scraper/cheerio-scraper/README.md [hunk 1]
```diff
@@ -246,19 +246,16 @@ visit the [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/J
 
 - ##### **`Apify: Object`**
 
-  A reference to the main namespace of [Apify SDK](https://sdk.apify.com/docs/api/apify).
+  A reference to the [Actor](https://apify.github.io/apify-sdk-js/api/apify/class/Actor) object from [Apify SDK](https://apify.github.io/apify-sdk-js/api/apify/).
   This is equivalent to:
 
   ```javascript
-  const Apify = require('apify');
+  const { Actor: Apify } = require('apify');
```
Comment (B4nan) [inline packages/actor-scraper/cheerio-scraper/README.md:253]: i would go for ESM examples, and use `Actor` directly without aliasing. wdyt @mnmkng?

Comment (mnmkng) [reply packages/actor-scraper/cheerio-scraper/README.md:253]: yes

**Comment by B4nan** [comment]
We have the new [SDK docs](https://sdk.apify.com) live, so I guess we should keep some of the links there that are now replaced with crawlee.dev or removed totally. 

(the links to API docs for crawlers etc are correct, that lives in crawlee.dev nowadays)

**Comment by barjin** [comment]
bumping this, AFAIK should be mergeable?

File: packages/actor-scraper/cheerio-scraper/README.md [hunk 2]
```diff
@@ -108,13 +108,13 @@ and the result is stored in the [`context.contentType`](#contenttype-object) obj
 The actor does not employ a full-featured web browser such as Chromium or Firefox, so it will not be sufficient for web pages that render their content dynamically using client-side JavaScript. To scrape such sites, you might prefer to use [**Web Scraper**](https://apify.com/apify/web-scraper) (`apify/web-scraper`), which loads pages in a full browser and renders dynamic content.
 
 Since Cheerio Scraper's **Page function** is executed in the context of the server, it only supports server-side code running in Node.js. If you need to combine client- and server-side libraries in Chromium using the [Puppeteer](https://github.com/GoogleChrome/puppeteer/) library, you might prefer to use
-[**Puppeteer Scraper**](https://apify.com/apify/puppeteer-scraper) (`apify/puppeteer-scraper`). For even more flexibility and control, you might develop a new actor from scratch in Node.js using the [Apify SDK](https://sdk.apify.com).
+[**Puppeteer Scraper**](https://apify.com/apify/puppeteer-scraper) (`apify/puppeteer-scraper`). For even more flexibility and control, you might develop a new actor from scratch in Node.js using [Cheerio](https://cheerio.dev).
```
Comment (B4nan) [inline packages/actor-scraper/cheerio-scraper/README.md:111]: should be crawlee here i guess? 
:] cheerio.dev does not even exist

**Review by B4nan — APPROVED** [review]
otherwise looking good, feel free to merge

File: packages/apify/package.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "bugs": {
         "url": "https://github.com/apify/apify-sdk-js/issues"
     },
-    "homepage": "https://apify.github.io/apify-sdk-js/",
+    "homepage": "https://sdk.apify.com/",
```
Comment (B4nan) [inline packages/apify/package.json:45]: nit: 
```suggestion
    "homepage": "https://sdk.apify.com",
```

**Pull request merged**